src/openrct2/object/WallObject.cpp

```diff
@@ -120,6 +120,7 @@ void WallObject::ReadJson(IReadObjectContext* context, json_t& root)
                 { "hasTernaryColour",       WALL_SCENERY_HAS_TERNARY_COLOUR,    Json::FlagType::Normal },
                 { "hasGlass",               WALL_SCENERY_HAS_GLASS,             Json::FlagType::Normal },
                 { "isBanner",               WALL_SCENERY_IS_BANNER,             Json::FlagType::Normal },
```

Comment:
although counterintuitive this line needs the WALL_SCENERY_IS_BANNER replace with DOUBLE_SIDED version

Reply:
Thank you for your help.